R/civis_ml.R

```diff
@@ -827,6 +829,14 @@ fetch_logs.civis_ml <- function(object, limit = 100, ...) {
   format_scripts_logs(logs)
 }
 
+#' @export
+fetch_logs.civis_ml_error <- function(object, limit = 100, ...) {
+  job_id <- attr(object, "args")$id
+  run_id <- attr(object, "args")$run_id
+  logs <- scripts_list_custom_runs_logs(job_id, run_id, limit = limit)
```

Comment:
I suppose that would hit the `error` branch of the `tryCatch` above on line 537.